e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -520,6 +520,24 @@ describe("scenarios > embedding > dashboard parameters", () => {
         H.assertSheetRowsCount(54)(sheet);
       },
     );
+
+    cy.log(
+      "The PDF download button should be clickable when there is no title, but has parameters (metabase#59503)",
+    );
+    cy.get("@dashboardId2").then((dashboardId) => {
+      const payload = {
+        resource: { dashboard: dashboardId },
+        params: {},
+      };
+      H.visitEmbeddedPage(payload, {
+        pageStyle: {
+          downloads: true,
+          titled: false,
+        },
+      });
+    });
+
+    cy.button("Download as PDF").should("be.visible").click();
```

Comment:
changed from `cy.findByTestId("export-as-pdf-button")` as it doesn't seem to exist on the release branch. But I should have used the label in the first place 🫠